js/Events/commonEvents.js

```diff
@@ -8,23 +8,16 @@ interviewApp.Events.EventsCommon = function () {
     };
 
     const AddEventsCommon = function () {
-        logOutButton.addEventListener('click', logOutListener);
+        logOutButton && logOutButton.addEventListener('click', logOutListener);
```

Comment:
Why the truthy check? Is it possible that `logOutButton` doesn't exist at this point?